addon/templates/components/render-content.hbs

```diff
@@ -3,5 +3,9 @@
 {{else if contentIsComponentDefinition}}
   {{component content}}
 {{else}}
-  {{content}}
+  {{#if hasBlock}}
```

Comment:
Any reason not to move this as an `else if` on the previous line?

Reply:
sure, why not